calico/reference/resources/felixconfig.mdx

```diff
@@ -124,7 +124,7 @@ spec:
 | bpfEnabled                         | Enable eBPF dataplane mode. eBPF mode has some limitations, see the [HOWTO guide](../../operations/ebpf/enabling-ebpf.mdx) for more details. | true, false                                                  | boolean                                 | false                                                        |
 | bpfDisableUnprivileged             | If true, Felix sets the kernel.unprivileged_bpf_disabled sysctl to disable unprivileged use of BPF. This ensures that unprivileged users cannot access Calico's BPF maps and cannot insert their own BPF programs to interfere with the ones that {{prodname}} installs. | true, false                                                  | boolean                                 | true                                                         |
 | bpfLogLevel                        | In eBPF dataplane mode, the log level used by the BPF programs. The logs are emitted to the BPF trace pipe, accessible with the command `tc exec bpf debug`. | Off,Info,Debug                                               | string                                  | Off                                                          |
-| bpfDataIfacePattern                | In eBPF dataplane mode, controls which interfaces Felix should attach BPF programs to catch traffic to/from the external network. This needs to match the interfaces that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to NodePorts and services from outside the cluster. It should not match the workload interfaces (usually named cali...).. | regular expression                                           | string                                  | ^(en&#46;&#42;&#124;eth&#46;&#42;&#124;tunl0$)               |
+| bpfDataIfacePattern                | In eBPF dataplane mode, controls which interfaces Felix should attach BPF programs to catch traffic to/from the external network. This needs to match the interfaces that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to NodePorts and services from outside the cluster. It should not match the workload interfaces (usually named cali...).. | regular expression                                           | string                                  | ^((en|wl|ww|sl|ib)[Popsx].*|(eth|wlan|wwan|bond).*|tunl0$|vxlan.calico$|vxlan-v6.calico$|wireguard.cali$|wg-v6.cali$) |
```

Comment:
Try this:

^((en&#124;wl&#124;ww&#124;sl&#124;ib)[Popsx].\*&#124;(eth&#124;wlan&#124;wwan&#124;bond).\*&#124;tunl0$&#124;vxlan.calico$&#124;vxlan-v6.calico$&#124;wireguard.cali$&#124;wg-v6.cali$)